tests/core/test_validation.py

```diff
@@ -15,7 +15,7 @@
     'nonce': 3,
 }
 
-TEST_PRIVATE_KEY = b'\0' * 32
+TEST_PRIVATE_KEY = b'\0' * 31 + b'\x01'
```

Comment:
Newer crypto libraries complain about using an all-zero private key (yay!). We still use a very unsafe key for tests, but not 0.